src/main/java/com/bet/Model/uniqueCard.java

```diff
@@ -0,0 +1,28 @@
+public class uniqueCard extends Bet{
+
+    private int cardNumbersQuantity;
+    private int[] numbers = new int[cardNumbersQuantity] ;
+
+    uniqueCard(BetType type, String[] bettors, Date date,int[] numbers, int cardNumbersQuantity){
+        super(type, bettors, date);
+        type=BetType.uniqueCard;
+        this.numbers=numbers;
+    }
+
+    public int getCardNumbersQuantity() {
+        return this.cardNumbersQuantity;
+    }
+
+    public void setCardNumbersQuantity(int cardNumbersQuantity) {
+        this.cardNumbersQuantity = cardNumbersQuantity;
+    }
+
+    public int[] getNumbers() {
+        return this.numbers;
+    }
+
+    public void setNumbers(int[] numbers) {
+        this.numbers = numbers;
+    }
+    
+}
```

Comment:
it's a convention that a class ends with a blank line so much so that github puts a warning at the end of your code.